Build/Newtonsoft.Json.nuspec

```diff
@@ -11,5 +11,34 @@
     <iconUrl>http://www.newtonsoft.com/content/images/nugeticon.png</iconUrl>
     <licenseUrl>https://raw.github.com/JamesNK/Newtonsoft.Json/master/LICENSE.md</licenseUrl>
     <tags>json</tags>
+    <dependencies>      
+      <group targetFramework="net2" />
+      <group targetFramework="sl5" />
+      <group targetFramework="portable-net40+sl5+wp80+win8+wpa81" />
```

Comment:
I tested this further and do not believe those groups are needed for sl5 or net2. It appears that the `dotnet` TFM dependencies are only evaluated for .NET 4.5+ projects (or more accurately, System.Runtime) based projects.

The case where the empty groups are needed is if you target .NET 4.6+ stuff (like the newer .NET Core packages that use System.Runtime 4.0.20 or higher). Then you need to block those dependencies from trying to be used by earlier platforms. Make sense?